### PR TITLE
General improvements in Personne Bundle

### PR DESCRIPTION
**Pull request by Olivier-R**
* I18n
* Responsivity
* Standard content between views
* Other fix & update

### CODE REVIEW

File: src/Mgate/PersonneBundle/Resources/views/Personne/listeDiffusion.html.twig [hunk 1]
```diff
@@ -45,13 +55,17 @@
...
-{{ cotisantsEtu }}
-</pre>
-<h3>Liste de diffusion cotisant ({{ nbrCotisants }}) (hors mails école)</h3>
-<pre>
-{{ cotisants }}
-</pre>
+    </table>
+</div>
+
+    <h3>Liste de diffusion cotisant ({{ nbrCotisantsEtu}})(mails école)</h3>
```
Comment (theofidry) [inline src/Mgate/PersonneBundle/Resources/views/Personne/listeDiffusion.html.twig]: The translations are missing here?

File: src/Mgate/PersonneBundle/Resources/views/Prospect/TabVoir/etudes.html.twig [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+<table class="table table-bordered table-striped dt-responsive text-left" id="etudes">
     <thead>
     <tr>
-        <th>Date d'ajout</th>
-        <th>Nom</th>
-        <th>Statut</th>
-        <th>Sujet</th>
-        <th>Montant HT (€)</th>
-        <th>JEHs</th>
+        <th>{{ 'study.fields.creation_date'|trans({}, 'person') }}</th>
```
Comment (theofidry) [inline src/Mgate/PersonneBundle/Resources/views/Prospect/TabVoir/etudes.html.twig]: maybe you could add your own twig filter to have a shortcut `strans('person')`

File: var/migrations/Version20170614232630.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('Update `AdminParam` set name = "anneeCreation" where name = "anneCreation"');
```
Comment (theofidry) [inline var/migrations/Version20170614232630.php]: It would probably be good to follow a proper SQL style:

```sql
UPDATE AdminParam SET name = 'anneeCreation' WHERE name = 'anneCreation';
```

(MySQL keywords in uppercase)

File: var/key_value_store/config.json [hunk 4]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+{"tva":0.2,"logo":"s:87:\"https:\/\/n7consulting.fr\/themes\/IOTH-bootstrap-basic\/assets\/images\/logo_n7consulting.png\";","adresse":"s:46:\"2 rue Charles Camichel,  31071 Toulouse CEDEX7\";","url":"s:23:\"https:\/\/n7consulting.fr\";","domaineEmailEtu":"s:15:\"etu.enseeiht.fr\";","domaineEmailAncien":"s:18:\"almuni.enseeiht.fr\";","anneeCreation":"s:4:\"1977\";","annee1Jeyser":"s:4:\"2015\";","presidentPrenom":"s:7:\"Antoine\";","presidentNom":"s:6:\"Dupond\";","presidentTexte":"s:15:\"Son Pr\u00e9sident,\";","tresorierPrenom":"s:6:\"Thomas\";","tresorierNom":"s:6:\"Dupont\";","tresorierTexte":"s:15:\"Son Tr\u00e9sorier,\";","nom":"s:13:\"N7 Consulting\";","abbr":"s:3:\"N7C\";","email":"s:23:\"contact@n7consulting.fr\";","gaTracking":null,"namingConvention":"s:3:\"nom\";"}
```
Comment (theofidry) [inline var/key_value_store/config.json]: if you have a `.dist` why is there still a not `.dist` one? Also I think this file should be formatted, it's unreable

File: app/Resources/views/layout.html.twig [hunk 5]
```diff
@@ -122,7 +122,7 @@ along with Incipio as the file LICENSE.  If not, see <http://www.gnu.org/license
             {# It is illegal (cf licence) to remove the link to github #}
             {# And dishonest to remove the reference to Jeyser's main contributors.#}
             <p class="text-muted" style="text-align: center">
-                <i>© Jeyser CRM 2017 &#124; N7 Consulting - AB</i>
+                <i>© Jeyser CRM 2017 &#124; M-GaTE - OR &#124; N7 Consulting - AB</i>
```
Comment (Stoakes) [inline app/Resources/views/layout.html.twig]: If we were only 2 contributors I would agree with that change.

However, if each and ever one of contributors add its letters, the footer will be too big. What about reduce it to 
`... M-GaTE | N7 Consulting` ?

File: app/config/config.yml [hunk 6]
```diff
@@ -35,6 +35,7 @@ twig:
     globals:
         role_voir_confidentiel: "%role_voir_confidentiel%"
         authorizedStorageSize: "%junior_authorizedstoragesize%"
+        technical_to: "%technical_to%"
```
Comment (Stoakes) [inline app/config/config.yml:38]: Would be better if we could avoid a global variable. 

~~`{{ app.environnment.technical_to }}` should work~~

File: app/Resources/TwigBundle/views/Exception/error500.html.twig [hunk 7]
```diff
@@ -9,9 +9,8 @@
 
             <div class="error-content">
                 <h3><i class="fa fa-warning text-red"></i> T'as tout cassé !</h3>
-
                 <p>
-                    Si le problème persiste, tu nous envoyer un petit rapport d'erreur à <a href="mailto:dsi@N7consulting.fr">dsi@N7consulting.fr</a> <br>
+                    Si le problème persiste, tu peux nous envoyer un petit rapport d'erreur à <a href="mailto:{{ technical_to }}">{{ technical_to }}</a><br>
```
Comment (Stoakes) [inline app/Resources/TwigBundle/views/Exception/error500.html.twig]: Sending an email to their own IT is ok, I think they would have think to it.

I think we should add a notice to where an other source of support can be found, Github or by email. WDYT ?

File: src/Mgate/SuiviBundle/Controller/EtudeController.php [hunk 8]
```diff
@@ -232,12 +229,22 @@ public function modifierAction(Request $request, Etude $etude)
...
-                return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
+                if ($request->get('ap')) {
+                    return $this->redirect($this->generateUrl('MgateSuivi_ap_rediger', array('id' => $etude->getId())));
+                } else {
+                    return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
+                }
+            } else {
+                $errors = $this->get('validator')->validate($etude);
+                foreach ($errors as $error) {
+                    array_push($error_messages, $error->getPropertyPath() . ' : ' . $error->getMessage());
```
Comment (Stoakes) [inline src/Mgate/SuiviBundle/Controller/EtudeController.php]: I would rather use flashbags `$this->addFlash()` than recreating the same mechanism with a custom array

File: src/Mgate/PersonneBundle/Resources/views/Employe/ajouter.html.twig [hunk 9]
```diff
@@ -8,33 +8,84 @@
...
+        <li><a href="{{ path('MgatePersonne_prospect_voir', {'id': prospect.id}) }}">{{ prospect.nom }}</a></li>
+        <li class="active">{{ 'employee.add_employee'|trans({}, 'person') }}</li>
     </ul>
 {% endblock %}
 
 {% block content_bundle %}
-
-    {% include "MgatePersonneBundle:Employe:formulaire.html.twig" %}
-
+    <div class="table-responsive content">
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Employe/ajouter.html.twig:31]: Why did you remove the a separate file for the form ?

Can you please keep it ? It reduces the duplicated code between edit and create view.

File: src/Mgate/TresoBundle/Form/Type/BVType.php [hunk 10]
```diff
@@ -35,7 +35,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('mission', Select2EntityType::class, array(
                       'label' => 'Mission',
                        'class' => 'Mgate\\SuiviBundle\\Entity\\Mission',
-                       'choice_label' => 'nom',
+                       'choice_label' => 'reference',
```
Comment (Stoakes) [inline src/Mgate/TresoBundle/Form/Type/BVType.php]: That commit basically remove changes done in 41283abb86da49c18cfe50d528f5cad0d1c51648

A compromise is to inject the namingConvention parameter in that form  (through an option, it will be ok) and use it as value of that form.  It has to be a robust solutions, thus add reference as default value in the OptionResolver

File: src/Mgate/PersonneBundle/Resources/translations/person.fr.yml [hunk 11]
```diff
@@ -0,0 +1,155 @@
+generic:
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/translations/person.fr.yml]: Adding translations is nice, but as long as that project is mainly french, translations keys should be in french. It makes main developers work much easier.

I am sorry to say that because I know it has been a big work for you, but would you mindchanging your work to  match the already existing conventions. You can have an example in [suivi translations](https://github.com/n7consulting/Incipio/blob/master/src/Mgate/SuiviBundle/Resources/translations/suivi.fr.yml)

File: src/Mgate/PersonneBundle/Resources/views/Prospect/index.html.twig [hunk 12]
```diff
@@ -32,13 +30,13 @@
             <i class="fa fa-plus"></i> Ajouter un prospect
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Prospect/index.html.twig]: Missing translation

File: src/Mgate/PersonneBundle/Resources/translations/person.fr.yml (hunk 11 above)
Comment (theofidry) [reply src/Mgate/PersonneBundle/Resources/translations/person.fr.yml]: That doesn't make sense though like for the code in French. It doesn't make sense when the language doesn't support well the language (here French). Either use generic translations keys (so English is the preferred choice) or don't use translations at all.

File: app/config/config.yml (hunk 6 above)
Comment (Olivier-R) [reply app/config/config.yml:38]: Thx, I was looking for something like that

File: src/Mgate/PersonneBundle/Resources/views/Personne/listeDiffusion.html.twig (hunk 1 above)
Comment (Olivier-R) [reply src/Mgate/PersonneBundle/Resources/views/Personne/listeDiffusion.html.twig]: As it's redundant, I thought to delete it

File: app/Resources/views/layout.html.twig (hunk 5 above)
Comment (Olivier-R) [reply app/Resources/views/layout.html.twig]: True, I'll reduce it

File: app/Resources/TwigBundle/views/Exception/error500.html.twig (hunk 7 above)
Comment (Olivier-R) [reply app/Resources/TwigBundle/views/Exception/error500.html.twig]: A link to issues ?

File: src/Mgate/PersonneBundle/Resources/views/Employe/ajouter.html.twig (hunk 9 above)
Comment (Olivier-R) [reply src/Mgate/PersonneBundle/Resources/views/Employe/ajouter.html.twig:31]: Because the last table line between the two views are slightly different. Should I move only the common rows to this separate file ?

File: src/Mgate/PersonneBundle/Resources/translations/person.fr.yml (hunk 11 above)
Comment (Stoakes) [reply src/Mgate/PersonneBundle/Resources/translations/person.fr.yml]: It does.

One day maybe, Jeyser will be an flourishing, multi country project. For the moment it is managed by 2 french juniors-entreprises, by frenchs developers for french users. 

If keys are in english, meaning is less obvious for the totality of developers involved on the project. 
It is not necessary to add additional difficulty to the project.
So ok it's not as standard as it could be, but isn't it better (as for french) to cope with `personne.filiere_modifier` rather than `people.pathways_edit` ?

File: app/Resources/TwigBundle/views/Exception/error500.html.twig (hunk 7 above)
Comment (Stoakes) [reply app/Resources/TwigBundle/views/Exception/error500.html.twig]: In addition of email to technical_to, yes good.

File: src/Mgate/PersonneBundle/Resources/translations/person.fr.yml (hunk 11 above)
Comment (Olivier-R) [reply src/Mgate/PersonneBundle/Resources/translations/person.fr.yml]: As stated in the readme, this is for all Junior Entreprises, and I think that other developpers will join easier this way (+ english is a must for devs)

Comment (Stoakes) [reply src/Mgate/PersonneBundle/Resources/translations/person.fr.yml]: English is easier for devs when they speak english. When they don't it is an additional hassle. It's not as if Jeyser was crumbling under contributors.

Code in english if you want, but translations key will be in french. 

Besides it will remove inconsistence in keys (`mail_valid`, `mail_invalid`, `subscribed_newsletter`, `study` instead of project, `mandate` instead of term)

**Comment by Stoakes** [comment]
I momentanly close that PR to make the dev branch consistent with master

**Comment by Stoakes** [comment]
At least the 
> Merge branch 'dev-mgate'

commit is making things looks bad

File: var/key_value_store/config.json (hunk 4 above)
Comment (Stoakes) [reply var/key_value_store/config.json]: It's one of my commit.
I know it's weird but It's a kind of compromise between code complexity and purity. You might have a suggestion.

Config.json.dist is the model of file, copied by the install script. 
At least one of the value stored in config.json is injected into I don't remember which service. Thus when an external service (insights, travis..) build the project, the config.json should be there.

With travis, I can easily copy the script before launching tests. However, for sensiolabs insights, it has to be done in a composer pre-install script. Every time the apache docker container is launched, it executes `composer install` (and thus execute pre-install script, so would copy the dist file on the existing config file). That behaviour would mean that each time the web container is started, configuration is lost.

My workaround is to distribute both file, keep install script minimal (no tests to check if the file exists and thus don't depend on the OS, no need to create additional side effects) and don't track changes on config.json.

Comment (theofidry) [reply var/key_value_store/config.json]: >That behaviour would mean that each time the web container is started, configuration is lost.

To me it sounds like it's your issue? Your install script should just copy the `.dist` if the non-dist doesn't exist, otherwise it shouldn't do anything.

Comment (Stoakes) [reply var/key_value_store/config.json]: Of course.  

That's why it is not the end of my message. I then explain that in order to keep things easy (and cross platform, because a bash script would not necessary work on windows) I kept the file in the index which is the most straight forward solution.

Comment (theofidry) [reply var/key_value_store/config.json]: You can go with a small PHP script like Incenteev does for the `parameters.yml`. Its then easy to hook on Composer and cross platform

File: src/Mgate/SuiviBundle/Controller/EtudeController.php [hunk 13]
```diff
@@ -232,12 +229,22 @@ public function modifierAction(Request $request, Etude $etude)
                 $em->persist($etude);
                 $em->flush();
 
-                return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
+                if ($request->get('ap')) {
+                    return $this->redirect($this->generateUrl('MgateSuivi_ap_rediger', array('id' => $etude->getId())));
+                } else {
+                    return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
+                }
+            } else {
```
Comment (Stoakes) [inline src/Mgate/SuiviBundle/Controller/EtudeController.php]: Unnecessary else (the condition above always return, so no need to add an else block around the following code)

Should be the same in `modifierAction`

File: src/Mgate/SuiviBundle/Controller/EtudeController.php [hunk 14]
```diff
@@ -164,11 +164,7 @@ public function addAction(Request $request)
                 $em->persist($etude);
                 $em->flush();
 
-                if ($request->get('ap')) {
-                    return $this->redirect($this->generateUrl('MgateSuivi_ap_rediger', array('id' => $etude->getId())));
-                } else {
-                    return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
-                }
+                return $this->redirect($this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom())));
```
Comment (Stoakes) [inline src/Mgate/SuiviBundle/Controller/EtudeController.php]: Maybe [`return $this->redirectToRoute()` ](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php#L99)

(Even though that `$this->redirect($this->generateUrl(` pattern is frequently used in the application, we should begin to replace it)

Comment (Olivier-R) [reply src/Mgate/SuiviBundle/Controller/EtudeController.php]: I'll open an issue for that, and replace them all in a single commit

File: src/Mgate/SuiviBundle/Controller/EtudeController.php (hunk 13 above)
Comment (theofidry) [reply src/Mgate/SuiviBundle/Controller/EtudeController.php]: You could also use a ternary operator:

```php
$redirectUrl = $request->get('ap')
    ? $this->generateUrl('MgateSuivi_ap_rediger', array('id' => $etude->getId()))
    : $this->generateUrl('MgateSuivi_etude_voir', array('nom' => $etude->getNom()))
;

return $this->redirect($redirectUrl);
```

Comment (Olivier-R) [reply src/Mgate/SuiviBundle/Controller/EtudeController.php]: It generates errors to be displayed when the submitted form is not valid

Comment (theofidry) [reply src/Mgate/SuiviBundle/Controller/EtudeController.php]: it's strictly equivalent to your code above...

File: app/config/config.yml (hunk 6 above)
Comment (Olivier-R) [reply app/config/config.yml:38]: @theofidry What do you think would best suit: 1) create a global variable 2) create a twig extension getenv that gives access to environment variables ?  N°1 may be more secure ?

Comment (theofidry) [reply app/config/config.yml:38]: I think a global variable like that is fine. It's only for twig so that's ok. An alternative would be to inject the container or the parameter bag as a global variable, but IMO this is less ideal as you expose way more than you should. I'm not aware of any other easy way to do this but to be fair I'm not a heavy twig user

Comment (Olivier-R) [reply app/config/config.yml:38]: Ok, then global variable

File: app/Resources/views/layout.html.twig [hunk 15]
```diff
@@ -122,7 +122,7 @@ along with Incipio as the file LICENSE.  If not, see <http://www.gnu.org/license
             {# It is illegal (cf licence) to remove the link to github #}
             {# And dishonest to remove the reference to Jeyser's main contributors.#}
             <p class="text-muted" style="text-align: center">
-                <i>© Jeyser CRM 2017 &#124; N7 Consulting - AB</i>
+                <i>© Jeyser CRM 2017 &#124; M-GaTE &#124; N7 Consulting</i>
```
Comment (Stoakes) [inline app/Resources/views/layout.html.twig]: Very minor but IMO it would be better if it was M-Gate &  N7 Consulting rather than M-Gate | N7 Consulting.

It returns the idea of cooperation and suits more with project philosophy

Comment (Olivier-R) [reply app/Resources/views/layout.html.twig]: Yes, done

File: src/Mgate/PersonneBundle/Resources/views/Membre/impayes.html.twig [hunk 16]
```diff
@@ -45,12 +46,17 @@
...
                 </td>
                 <td>{{ membre.personne ? membre.personne.mobile }}</td>
-                <td>{{ membre.personne ? membre.personne.adresse }} {{ membre.personne ? membre.personne.codepostal }} {{ membre.personne ? membre.personne.ville }}</td>
+                <td>
+                    {{ membre.personne ? membre.personne.adresse }}<br/>
+                    {{ membre.personne ? membre.personne.codepostal }} {{ membre.personne ? membre.personne.ville }}
+                </td>
             </tr>
+        {% else %}
+            <li>{{ 'member.empty_unpaid_list_description'|trans({}, 'person') }}</li>
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Membre/impayes.html.twig]: `<tr><td colspan="6">Pas d'impayés</td></tr>` instead of `<li>`.

Not sure it is even necessary because datatables will display an empty table message.

File: src/Mgate/PersonneBundle/Resources/views/Membre/index.html.twig [hunk 17]
```diff
@@ -53,16 +53,21 @@
...
                 <td>{{ membre.personne ? membre.personne.email }}</td>
                 <td>{{ membre.personne ? membre.personne.mobile }}</td>
-                <td>{{ membre.personne ? membre.personne.adresse }} {{ membre.personne ? membre.personne.codepostal }} {{ membre.personne ? membre.personne.ville }}</td>
+                <td>
+                    {{ membre.personne ? membre.personne.adresse }}<br/>
+                    {{ membre.personne ? membre.personne.codepostal }} {{ membre.personne ? membre.personne.ville }}
+                </td>
             </tr>
+        {% else %}
+            <li>{{ 'member.empty_member_list_description'|trans({}, 'person') }}</li>
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Membre/index.html.twig]: `tr & td` rather than `li`

File: src/Mgate/SuiviBundle/Resources/views/Etude/modifier.html.twig [hunk 18]
```diff
@@ -10,7 +10,11 @@
 
 
 {% block content_bundle %}
-
+    {% for error in errors %}
```
Comment (Stoakes) [inline src/Mgate/SuiviBundle/Resources/views/Etude/modifier.html.twig]: I have `Variable "errors" does not exist.` when i want to modify an etude

File: src/Mgate/PersonneBundle/Resources/views/Poste/index.html.twig [hunk 19]
```diff
@@ -1,26 +1,32 @@
...
+        <li class="active">{{ 'dashboard.postes_filieres'|trans({}, 'dashboard') }}</li>
+    </ul>
+{% endblock %}
 
 {% block content_bundle %}
-
     <div class="row">
         <div class="col-sm-6 col-xs-12">
-            <h2>Liste des Postes</h2>
+            <h2>{{ 'poste.liste_postes'|trans({}, 'personne') }}</h2>
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Poste/index.html.twig]: Missing translation key

File: app/Resources/TwigBundle/views/Exception/error500.html.twig [hunk 20]
```diff
@@ -9,10 +9,10 @@
 
             <div class="error-content">
                 <h3><i class="fa fa-warning text-red"></i> T'as tout cassé !</h3>
-
                 <p>
-                    Si le problème persiste, tu nous envoyer un petit rapport d'erreur à <a href="mailto:dsi@N7consulting.fr">dsi@N7consulting.fr</a> <br>
-                    Explique bien le contexte et mets des petites images et tout et tout ;)
+                    Si le problème persiste, tu peux nous envoyer un petit rapport d'erreur à <a href="mailto:{{ technical_to }}">{{ technical_to }}</a><br>
+                    Pour que le problème soit rapidement pris en compte et traité, le mieux serait de nous faire un rapport directement sur <a href="https://github.com/n7consulting/Incipio/issues/new">GitHub</a></br>
```
Comment (Stoakes) [inline app/Resources/TwigBundle/views/Exception/error500.html.twig]: `<br>` or `<br />` but not `</br>`

Problem common to the different error views

File: src/Mgate/PersonneBundle/Resources/views/Prospect/formulaire.html.twig [hunk 21]
```diff
@@ -1,45 +1,23 @@
...
+                <th>{{ 'champs.code_postal'|trans({}, 'personne') }}</th>
+                <td>{{ form_widget(form.codepostal) }} {{ form_errors(form.codepostal) }}</td>
+            </tr>
+            <tr>
+                <th>{{ 'champs.ville'|trans({}, 'personne') }}</th>
+                <td>{{ form_widget(form.ville) }} {{ form_errors(form.ville) }}</td>
+            </tr>
+                <th>{{ 'champs.pays'|trans({}, 'personne') }}</th>
+                <td>{{ form_widget(form.pays) }} {{ form_errors(form.pays) }}</td>
+            </tr>
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Prospect/formulaire.html.twig]: closing tag matches nothing

File: src/Mgate/PersonneBundle/Resources/views/Prospect/voir.html.twig [hunk 22]
```diff
@@ -12,35 +12,32 @@
...
+        <li class="active">{{ prospect.nom }}</li>
     </ul>
 {% endblock %}
 
 {% block content_bundle %}
-
     <h2>{{ prospect.nom }}</h2>
-    <em><h4>Type de prospect : {{ prospect.EntiteToString }}</h4></em>
+    <em><h4>{{ 'prospect.champs.type'|trans({}, 'personne') }} : {{ prospect.EntiteToString }}</h4></em>
     <p>
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Prospect/voir.html.twig]: Not from you but : `<p>` around h4 is useless

File: src/Mgate/PersonneBundle/Resources/views/Prospect/TabVoir/employes.html.twig [hunk 23]
```diff
@@ -1,28 +1,31 @@
-<h3>Liste des employés</h3>
+<h3>{{ 'employee.liste_employe'|trans({}, 'personne') }}</h3>
```
Comment (Stoakes) [inline src/Mgate/PersonneBundle/Resources/views/Prospect/TabVoir/employes.html.twig]: missing translation key

**Review by Stoakes — CHANGES_REQUESTED** [review]

File: src/Mgate/PersonneBundle/Resources/views/Membre/impayes.html.twig (hunk 16 above)
Comment (Olivier-R) [reply src/Mgate/PersonneBundle/Resources/views/Membre/impayes.html.twig]: Right, I looked at the demo with an empty list. But I keep it for Poste and Filiere

File: src/Mgate/TresoBundle/Form/Type/BVType.php (hunk 10 above)
Comment (Stoakes) [reply src/Mgate/TresoBundle/Form/Type/BVType.php]: Any comment on that point ?

**Review by Stoakes — APPROVED** [review]
